docs/ARCHITECTURE.md

```diff
@@ -1,5 +1,11 @@
 [Back to Guides](README.md)
 
+This document focuses on architecture the 0.10.x version of ActiveModelSerializers. If you are interested in the architecture of the 0.8 or 0.9 versions,
+please refer to the [0.8 README](https://github.com/rails-api/active_model_serializers/blob/0-8-stable/README.md) or
+[0.9 README](https://github.com/rails-api/active_model_serializers/blob/0-9-stable/README.md).
+
+The original design is also available [here](https://github.com/rails-api/active_model_serializers/blob/0-9-stable/DESIGN.textile).
```

Comment:
I'd rather link to one of the original docs 

- `[serializers.textile in Rails (before reverted)](https://github.com/rails/rails/blob/fcacc6986ab60f1fb2e423a73bf47c7abd7b191d/railties/guides/source/serializers.textile)` and
- `[README.textile in first 0.0.1 commit](https://github.com/rails-api/active_model_serializers/commit/d72b66d4c5355b0ff0a75a04895fcc4ea5b0c65e?short_path=8aa641a#diff-8aa641a619b8ebeca0116d41774daf45)`